### PR TITLE
[8.0][MIGR] procurement_purchase_no_grouping from odoomrp/odoomrp-wip

### PR DESCRIPTION
**Pull request by tafaRU**
This PR supersedes https://github.com/OCA/purchase-workflow/pull/209.

### CODE REVIEW

File: procurement_purchase_no_grouping/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 OdooMRP team
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Procurement Purchase No Grouping",
+    "version": "8.0.1.0.0",
+    "author": "OdooMRP team, AvanzOSC, Pedro M. Baeza, "
```
Comment (pedrobaeza) [inline procurement_purchase_no_grouping/__openerp__.py]: Please change my name by Tecnativa, and you can remove `OdooMRP team`, as it's redundant.

**Comment by tafaRU** [comment]
Same errors appear at https://travis-ci.org/OCA/purchase-workflow/builds/204641899 so I assume they are unrelated to this module.

File: procurement_purchase_no_grouping/__openerp__.py (hunk 1 above)
Comment (tafaRU) [reply procurement_purchase_no_grouping/__openerp__.py]: > and you can remove OdooMRP team

What about the headers?

Comment (pedrobaeza) [reply procurement_purchase_no_grouping/__openerp__.py]: The same

**Comment by rubencabrera** [comment]
I added the changes from odoomrp-wip PR #1352 as a PR to @tafaRU 's branch to make them visible in this PR. I don't know if that's the proper way to go, maybe a bit too complicated, but hope it helps.

**Comment by tafaRU** [comment]
Hi @rubencabrera, please see https://github.com/tafaRU/purchase-workflow/pull/1#issuecomment-288027963

**Comment by tafaRU** [comment]
I'm not interested in continuing to work on this. So I close it.

**Pull request closed without merging**